com.unity.ml-agents/Runtime/ActionMasker.cs

```diff
@@ -110,7 +74,7 @@ public void SetActionMask(int branch, IEnumerable<int> actionIndices)
         /// </summary>
         /// <returns>A mask for the agent. A boolean array of length equal to the total number of
         /// actions.</returns>
-        internal bool[] GetMask()
+        internal bool[] GetDiscreteActionMask()
```

Comment:
I'm OK leaving this as GetMask(). The "DiscreteAction" part in the class name seems like enough

Reply:
You think? Why is it different from `SetMask()`?

Reply:
I just think it's redundant. `DiscreteActionMasker.GetMask()` and `DiscreteActionMasker.GetDiscreteActionMask()` convey the same amount of info. (same for `Set...`)